evaluation/benchmarks/run.seq.sh

```diff
@@ -143,11 +140,11 @@ max-temp(){
   cd max-temp/
 
   mkdir -p "$outputs_dir"
-
+  export IN=
```

Comment:
I am confused because in some benchmarks we get the variable from `setup.sh` and then `source_var`, and in other we export it here.

Reply:
I saw it in one more place, I think `oneliners`.